src/data/directory_content.rs

```diff
@@ -25,10 +43,65 @@ pub struct DirectoryEntry {
 impl DirectoryEntry {
     /// Creates a new directory entry from a path
     pub fn from_path(config: &FileDialogConfig, path: &Path) -> Self {
+        let mut size = None;
+        let mut last_modified = None;
+        let mut created = None;
+        let mut file_type = None;
+
+        #[cfg(feature = "info_panel")]
```

Comment:
In the end `other_meta_data` is set to `other_data` when building without the feature we cannot set HashMap to IndexMap